packages/core/src/prompts/lesson-assistant/old.ts

```diff
@@ -0,0 +1,742 @@
+// This is the original version of the prompt before
```

Comment:
When to delete?

Reply:
No, let's keep because there is a test on the diff between this prompt and the newly split up one. I'd like that in git history.